terraform/main.tf

```diff
@@ -16,6 +16,10 @@ terraform {
       source  = "germanbrew/dotenv"
       version = "~> 1.1.4"
     }
+     helm = {
```

Comment:
Is helm a must / or recommended? (looking for ways on who we can simplify setup)

Reply:
It is not a must but the easiest way to install Agones with Terraform.

Reply:
thanks